README.md

```diff
@@ -117,6 +209,95 @@ but this can be changed (per resolver instance) like so:
 val resolver = ArtifactResolver(cacheDir = fs.getPath("/some/cache/dir"))
 ```
 
+### Proxies and fetching
+
+Maven Archeologist uses OkHttp 4.0 and HttpArtifactFetcher can be created with a lambda that
+supplies a configured `OkHttpClient`. While this can be overridden by any lambda that takes
+a URL string and returns an `OkHttpClient` the default mechanism is sensitive to two environment
+variables ("HTTP_PROXY" and "NO_PROXY")
+
+#### Default Environment Variables
+
+> Environment variables are used in the default configuration, as there is often a need to have
+> a different context in CI without altering code, and maven resolution is often required in build
+> systems used in CI
+
+To use a proxy in the default setup, set the a variable in the environment in which the app you
+wish to use will run with the url of the proxy service (including, optionally, a
+username/password). e.g.:
+
+```sh
+setenv HTTP_PROXY=localhost:8080
```

Comment:
shouldn't this be export HTTP_PROXY=...?

Reply:
Probably.